eclass/waf-utils.eclass

```diff
@@ -34,6 +34,9 @@ esac
 waf-utils_src_configure() {
 	debug-print-function ${FUNCNAME} "$@"
 
+	# WAF looks for PKGCONFIG not PKG_CONFIG.
+	export PKGCONFIG=$(tc-getPKG_CONFIG)
```

Comment:
Please move this below `tc-export` below, and make it `local -x` if it doesn't need to be preserved for next phases.